integration/test-mb/test-amqp/src/test/java/org/wso2/ei/mb/test/amqp/BrokerTest.java

```diff
@@ -55,8 +56,8 @@ public void init() throws Exception {
             try {
                 String tempCarbonHome = serverManager.setupServerHome(archiveFilePath);
 
-                // load client configs to map
-                configurationReader = new ConfigurationReader();
+                // load client configs to map with valid username/password
+                configurationReader = new ConfigurationReader(false);
```

Comment:
Passing a boolean is not very readable. Shall we use a static factory method for this?

```java
configurationReader = ConfigurationReader.getClientConfigForInvalidUser();
```

Reply:
+1. Will update it.

Reply:
Fixed with 2ded599.